client/src/routes.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import {  Route,  ReactLocation } from "react-location";
+import AboutUs from "./pages/AboutUs";
+import Contact from "./pages/Contact";
+import Home from "./pages/Home";
+
+
+export const routes: Route[] = [
+
+{
+        path:"/home",
```

Comment:
Hey @Kanika637, just make `home ` route as default route, else looking good to me, good work 👍

Reply:
Do I have to change the name of the client to DoctorVerse? @naveen8801